db/migrate/20150526082930_add_rejection_information_to_organization.rb

```diff
@@ -0,0 +1,6 @@
+class AddRejectionInformationToOrganization < ActiveRecord::Migration
+  def change
+    add_column :organizations, :rejected, :boolean, default: false, null: false
+    add_column :organizations, :rejected_reason, :string
+  end
+end
```

Comment:
Should we maybe merge these additions into the migrations where we add the tables?